src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlManagedDatabase.md

```diff
@@ -0,0 +1,193 @@
+ResourceGroupName        : resourcegroup01
+ManagedInstanceName      : managedInstance1
+Location                 : westcentralus
+Id                       : /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/resourcegroup01/providers/Microsoft.Sql/managedInstances/managedInstance1/databases/managedDatabase1
+Name                     : managedDatabase1
+Tags                     :
+Collation                : SQL_Latin1_General_CP1_CI_AS
+Status                   : Online
+CreationDate             : 4/27/2018 2:30:07 PM
+EarliestRestorePoint     : 4/27/2018 2:40:47 PM
+RestorePointInTime       :
+DefaultSecondaryLocation : West US 2
+CatalogCollation         :
+CreateMode               :
+StorageContainerUri      :
+StorageContainerSasToken :
+SourceDatabaseId         :
+FailoverGroupId          :
+```
+
+This command gets a database named managedDatabase1 from a managed instance named managedInstance1.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ManagedDatabaseName
+The name of the Azure SQL Managed Database to retrieve.
+
+```yaml
+Type: String
+Parameter Sets: GetManagedDatabaseFromInputParameters
+Aliases: Name
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ManagedInstanceName
+The Azure Sql Managed Instance name.
+
+```yaml
+Type: String
+Parameter Sets: GetManagedDatabaseFromInputParameters
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+The name of the resource group.
+
+```yaml
+Type: String
+Parameter Sets: GetManagedDatabaseFromInputParameters
+Aliases:
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceId
+The resource id of Managed Database object to get
+
+```yaml
+Type: String
+Parameter Sets: GetManagedDatabaseFromAzureResourceId
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### System.String
+
+
+## OUTPUTS
+
+### System.Object
```

Comment:
You might have to manually update all of these outputs now that you have generated them with this System.Object.  Please update so they are all correct.